ocaml/libs/tracing/tracing.ml

```diff
@@ -696,36 +721,44 @@ module Export = struct
         with e -> Error e
     end
 
-    let export_to_endpoint traces endpoint =
+    let export_to_endpoint parent traces endpoint =
       debug "Tracing: About to export" ;
       try
-        File.with_stream (fun file_export ->
-            let export =
-              match endpoint with
-              | Url url ->
-                  Http.export ~url
-              | Bugtool ->
-                  file_export
-            in
-            Ok ()
-            |> Hashtbl.fold
-                 (fun _ spans result ->
-                   Content.Json.Zipkinv2.content_of spans
-                   |> export
-                   |> Result.fold ~ok:(fun () -> result) ~error:Result.error
-                 )
-                 traces
-            |> Result.iter_error raise
+        with_tracing ~parent ~name:"Tracing.export_to_endpoint" (fun parent ->
```

Comment:
the new let@ trick can help with minimizing the changeset by keeping indentation the same :)

`let@ parent = with_tracing ~parent ~name:... in`

Reply:
If this new, will I be able to do this now or wait until 4.14?

Reply:
this is not included in ocaml 4.14, it needs to be defined: `let ( let@ ) f x = f x`